core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -382,16 +388,22 @@ protected void startServingRPCServer() {
     } catch (TTransportException e) {
       throw new RuntimeException(e);
     }
+    // create executor service for thrift.
+    // use shutdown timeout as keep-alive.
+    int stopTimeoutMs = (int) Configuration.getMs(PropertyKey.MASTER_THRIFT_SHUTDOWN_TIMEOUT);
```

Comment:
NIT: rename to `shutdownTimeoutMs`

Reply:
I think. stop would make more sense as it's what thrift calls it. Changed the doc.